platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/pipeline/VirtualPipeline.java

```diff
@@ -477,20 +477,18 @@ private long currentTotalSize() {
     }
 
     /**
-     * Try to flush a copy. Hash it if necessary. If the copy is not flushed, it
-     * will be eligible to merge.
+     * Try to flush a copy. Hash it if necessary.
      *
      * @param copy the copy to flush
-     * @return if the copy was flushed
      */
-    private boolean tryFlush(final VirtualRoot<K, V> copy) {
+    private void flush(final VirtualRoot<K, V> copy) {
         if (copy.isFlushed()) {
             throw new IllegalStateException("copy is already flushed");
         }
         if (!copy.isHashed()) {
             hashCopy(copy);
         }
-        return copy.flush();
+        copy.flush();
```

Comment:
Changes related to the flush backlog were not reverted.

Reply:
Yes, flush backlog is no longer used, this is dead code that can be removed. No value in reverting